conf/modules.config

```diff
@@ -286,6 +286,16 @@ process {
         ]
     }
 
+    withName: BRACKEN_BRACKEN {
+        errorStrategy = 'ignore'
```

Comment:
From self-experience?

Reply:
When all reads are unclassified or there's nothing to redistribute, Bracken will error instead of exiting gracefully. So yes, it's needed based on experience. Unless we analyze kraken2 output and filter channels based on that.